src/common.rs

```diff
@@ -1,10 +1,10 @@
-use num_traits::{FromPrimitive, Signed};
+use num_traits::{FromPrimitive, Signed, Float, FloatConst};
 use std::fmt::Debug;
 
 /// Generic floating point number, implemented for f32 and f64
-pub trait FftNum: Copy + FromPrimitive + Signed + Sync + Send + Debug + 'static {}
+pub trait FftNum: Copy + FromPrimitive + Signed + Sync + Send + Debug + Float + FloatConst + 'static {}
```

Comment:
As far as I'm aware this is a breaking change, so unfortunately we will not be able to add these trait bounds.

Reply:
Yes this will cause trouble for anyone who has implemented FftNum for a type that doesn't implement the Float trait (which contains quite a lot of stuff).
I came to think about this one: https://github.com/ejmahler/RustFFT/pull/7